src/App/Fossa/Container/Analyze.hs

```diff
@@ -23,10 +19,7 @@ import Srclib.Types (parseLocator)
 
 analyzeMain :: ScanDestination -> Severity -> OverrideProject -> ImageText -> IO ()
 analyzeMain scanDestination logSeverity override image = withDefaultLogger logSeverity $ do
-  result <- runDiagnostics $ analyze scanDestination override image
-  case result of
-    Left err -> logError (renderFailureBundle err)
```

Comment:
Is this the culprit for why this PR was necessary?

Reply:
Yep.